components/schema-migrator/main.go

```diff
@@ -96,8 +129,58 @@ func main() {
 	}
 
 	if err != nil {
-		fmt.Printf("Error during migration, %s\n", err)
+		return fmt.Errorf("during migration: %s", err)
+	}
+
+	return nil
+}
+
+func quitCloudSqlProxy() error {
```

Comment:
nit - more suitable would be to name it killCloudSqlProxy since by killing we make it quit (and CloudSqlProxy does not quit itself).